DESCRIPTION

```diff
@@ -2,7 +2,7 @@ Type: Package
 Package: EpiNow2
 Title: Estimate Real-Time Case Counts and Time-Varying
     Epidemiological Parameters
-Version: 1.3.6.9007
+Version: 1.3.6.9008
```

Comment:
I have decided this is OTT if we want to stop doing this?

Reply:
Sounds good to me. Whilst we're there shouldn't we really follow version 1.3.5 by dev version 1.3.5.9000 so that we don't pre-determine whether the next release will be major, minor or patch? That's what `usethis::use_dev_version()` does.

Reply:
yes agree